src/Npgsql/NpgsqlActivitySource.cs

```diff
@@ -108,23 +108,26 @@ internal static void ReceivedFirstResponse(Activity activity, NpgsqlTracingOptio
     internal static void CommandStop(Activity activity)
     {
         activity.SetTag("otel.status_code", "OK");
+        activity.SetStatus(ActivityStatusCode.Ok);
```

Comment:
Is this a dup of the line just above? Or are there two different things? Same below for exception.

Reply:
Not exactly. `Activity.Status` (and `Activity.Description`) do not add any tags, they're essentially just fields in `Activity` and it's up to exporters to populate tags depending on `Activity.Status`. So in case there's an exporter which doesn't react on `Activity.Status`, then these tags will never be added if we remove them from there.
I'm more-or-less OK with removing explicit tags.

Reply:
I don't know enough about this... What do ASP.NET do, for instance?

Reply:
From what I've been able to fine, they just call `SetStatus`.

https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/61164064e25a111ad141d1f0cf2c8d329e16683a/src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs#L263-L266

Reply:
Thanks for looking - I guess we can do the same then?

Reply:
Yeah, sure